its/projects/NoSonarTest/Class1.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace CSLib.foo
+﻿[assembly: System.Reflection.AssemblyVersion("1.0.0")]
```

Comment:
This one is funny in a bitter way: `sonar.msbuild.testProjectPattern` marked project as a MAIN for S4MSB, but analyzer considered it as a test anyway. So only S101 was triggered (partially lucky), because of it's MAIN+TEST scope.

Changing analyzer implementation cleaned up the air and S3904 started to raise. I added this line to fix it and keep the assertions clean & focused.